app/main.py

```diff
@@ -1,6 +1,41 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+from datetime import datetime, timedelta
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+URL = "https://api.openweathermap.org/data/2.5/weather"
+CITY = "Paris"
+API_KEY = os.environ.get("WEATHER_API_KEY")
+
+
+def get_weather(
+        url: str = URL,
+        city: str = CITY,
+        api_key: str = API_KEY
+) -> None:
+    response = requests.get(url,
+                            params={"q": city,
```

Comment:
it's better to move it to variable for better readability, you could have 15 params